### DISCUSSION TITLE
Reopen the issue #1037

### DISCUSSION THREAD

**Discussion by prithvipatl** [fastapi/fastapi#7385] [Questions]
Having similar problem, where I want to just input the access_token to the authorization field in the header but the swagger UI does not use it while accessing the API.
Tagging the issue #1037

**Comment by tiangolo** [comment]
I'm sorry, I can't help you without more information. Please keep and follow the issue template. If you still have problems, please create a new issue following the issue template, step by step.

Given that, I'm gonna close this one.
